assets/app/scripts/controllers/otherResources.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ServicesController
+ * @description
+ * # ProjectController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('OtherResourcesController', function ($routeParams, $scope, AlertMessageService, DataService, ProjectsService, $filter, LabelFilter, Logger, APIService ) {
+    $scope.projectName = $routeParams.project;
+    $scope.labelSuggestions = {};
+    $scope.alerts = $scope.alerts || {};
+    $scope.emptyMessage = "Select a resource from the list above ...";  
+    $scope.kindSelector = {disabled: true};
+    $scope.kinds = _.filter(APIService.availableKinds(), function(kind) {
+      switch (kind.kind) {
+        case "ReplicationController":
+        case "DeploymentConfig":
+        case "BuildConfig":
+        case "Build":
+        case "Pod":
+        case "PersistentVolumeClaim":
+        case "Event":
+        case "Service":
+        case "Route":
+        case "ImageStream":
+        case "ImageStreamTag":
+        case "ImageStreamImage":
+        case "ImageStreamImport":
+        case "ImageStreamMapping":
+        case "Deployment":
+        case "LimitRange":
+        case "ResourceQuota":
+          return false;
```

Comment:
this particular list is the list of things we already show somewhere else in the UI, so there wasn't a reason to have them here

we could make the other blacklist overwriteable